### DISCUSSION TITLE
Purpose of "Difficult?" check box when verifying images.

### DISCUSSION THREAD

**Discussion by NeoDuder** [blakeblackshear/frigate#7969] [Ask A Question]
I've been going through image verification on my Frigate+ dashboard in order to fine tune my model. I'm curious about the purpose of the "Difficult?" check box. What does this actually do? Is it asking if the identification is difficult for me? or do I think it would be difficult for the AI? Currently I've only been checking it when an object is heavily obscured behind something, but would love some specific guidance and info on what effect this has?

**Comment by NickM-27** [comment]
See https://github.com/blakeblackshear/frigate/discussions/7932#discussioncomment-7099646

> currently difficult isn't used for anything but it is useful to have that data. It is meant to be used when you'd be surprised that the model would detect something, for example when a person is behind a tree or a car is partially occluded by another car. It is a bit subjective.
>
> The idea is that it could be used for example to exclude the image from the training set but use it in validation or other things as well.
